lib/marginalia/comment.rb

```diff
@@ -26,9 +26,20 @@ def self.construct_comment
         end
       end
       ret.chop!
+      ret = self.escape_sql_comment(ret)
       ret
     end
 
+    def self.escape_sql_comment(str)
+      if str.include?('/*') || str.include?('*/')
+        str = str.dup
+      end
```

Comment:
You are right, we don't need the `dup`. gsub should already create a new string for us.